Makefile

```diff
@@ -7,21 +7,23 @@ import:
 	psql --single-transaction --dbname pianoforte --file tmp/boundary.sql
 	ogr2ogr --config PG_USE_COPY YES -lco GEOMETRY_NAME=geometry -lco DROP_TABLE=IF_EXISTS -f PGDump tmp/city.sql data/city.csv -select name,'name:en','name:fr','name:ar',capital,type -nln city -oo X_POSSIBLE_NAMES=Lon* -oo Y_POSSIBLE_NAMES=Lat* -oo KEEP_GEOM_COLUMNS=NO -a_srs EPSG:4326
 	# Remove transaction management, as it does not cover the DROP TABLE; we'll cover the transaction manually with "psql --single-transaction"
-	sed --in-place '/BEGIN;/d' tmp/city.sql
-	sed --in-place '/END;/d' tmp/city.sql
-	sed --in-place '/COMMIT;/d' tmp/city.sql
+	sed -i.bak '/BEGIN;/d' tmp/city.sql
+	sed -i.back '/END;/d' tmp/city.sql
+	sed -i.back '/COMMIT;/d' tmp/city.sql
+	rm tmp/city.sql.back
 	psql --single-transaction --dbname pianoforte --file tmp/city.sql
 	ogr2ogr --config PG_USE_COPY YES -lco GEOMETRY_NAME=geometry -lco DROP_TABLE=IF_EXISTS -f PGDump tmp/country.sql data/country.csv -select name,'name:en','name:fr','name:ar' -nln country -oo X_POSSIBLE_NAMES=Lon* -oo Y_POSSIBLE_NAMES=Lat* -oo KEEP_GEOM_COLUMNS=NO -a_srs EPSG:4326
 	# Remove transaction management, as it does not cover the DROP TABLE; we'll cover the transaction manually with "psql --single-transaction"
-	sed --in-place '/BEGIN;/d' tmp/country.sql
-	sed --in-place '/END;/d' tmp/country.sql
-	sed --in-place '/COMMIT;/d' tmp/country.sql
+	sed -i.back '/BEGIN;/d' tmp/country.sql
+	sed -i.back '/END;/d' tmp/country.sql
+	sed -i.back '/COMMIT;/d' tmp/country.sql
+	rm tmp/country.sql.back
 	psql --single-transaction --dbname pianoforte --file tmp/country.sql
 update:
 	env PGHOST=/var/run/postgresql/ imposm3 run -config imposm.conf
 boundary:
 	python scripts/make_boundaries.py process
-	ogr2ogr --config PG_USE_COPY YES -lco GEOMETRY_NAME=geometry -lco DROP_TABLE=IF_EXISTS -f PGDump tmp/boundary.sql data/boundary.json -select name,'name:en','name:fr','name:ar','name:es','name:de','name:ru' -nln itl_boundary
+	ogr2ogr --config PG_USE_COPY YES -lco GEOMETRY_NAME=geometry -lco DROP_TABLE=IF_EXISTS -f PGDump tmp/boundary.sql data/boundary.json -select name,'name:en','name:fr','name:ar','name:es','name:de','name:ru','ISO3166-1:alpha2' -nln itl_boundary
```

Comment:
I suggest we rename the column is "iso" here: `'ISO3166-1:alpha2' AS iso" should work.

Reply:
100% for, I didn't know how to face it and didn't have the time to (or didn't want to, I don't know) read the doc :)

👍

Reply:
Unfortunately it doesn't seem to work 

And this with different ways to quote.

I obtain : `ERROR 1: Couldn't fetch requested layer 'AS'!`

Reply:
humm…
Have you tried `as`?

Reply:
http://www.gdal.org/ogr_sql.html#ogr_sql_fname_alias

Reply:
Seems to work as expected with :
```ogr2ogr --config PG_USE_COPY YES -lco GEOMETRY_NAME=geometry -lco DROP_TABLE=IF_EXISTS -f PGDump tmp/boundary.sql data/boundary.json -sql 'SELECT name,"name:en","name:fr","name:ar","name:es","name:de","name:ru","ISO3166-1:alpha2" AS iso FROM OGRGeoJSON' -nln itl_boundary```